src/main/java/ie/ait/bteam/drcare/rest/controller/UserRestController.java

```diff
@@ -0,0 +1,65 @@
+package ie.ait.bteam.drcare.rest.controller;
+
+import ie.ait.bteam.drcare.rest.exceptions.EntityNotFound;
+import ie.ait.bteam.drcare.rest.model.ResponseDTO;
+import ie.ait.bteam.drcare.rest.model.UserDTO;
+import ie.ait.bteam.drcare.rest.service.UserRestService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.validation.BindingResult;
+import org.springframework.validation.Errors;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@Controller
```

Comment:
I don't think there is need for Controller anymore

Reply:
The @Controller annotation is for REST api working with @RequestMapping

Reply:
That's what the @RestController is for @adheli